cmds/exp/uefiboot/README.md

```diff
@@ -0,0 +1,19 @@
+# How to build a UEFI payload
+
+*   Obtain edk2
+
+```shell
+git clone --branch uefipayload --recursive https://github.com/chengchiehhuang/edk2 uefipayload
```

Comment:
can you briefly describe how the linuxboot payload target is different from the coreboot one in edk2?

i also see a commit adding a whole IntelVTd DXE. is that not something standardized?

Reply:
Existing payload are more laptop-oriented. That's why it also got eMMC stuff and VTd is more server-oriented stuff.  It's coming from Edk2Platforms (https://github.com/tianocore/edk2-platforms).